app/assets/javascripts/mortgage_calculator/angular/directives/slider.js

```diff
@@ -123,16 +123,23 @@ App.directive('uiSlider', function() {
     angular.extend(options, expression);
     element.slider(options).on('slide', options.slide);
 
-    //Reconfigre slider when input blurs
+    $('.stamp-duty__submit').on('click', function() {
```

Comment:
Minor point, but I think we should avoid the jQuery selector here as it's an Angular directive and if we ever did want to use it again, this necessitates jQuery. If you use `angular.element` instead I believe this will do the trick (see line 8)